ZkLobbyServer/autohost/Commands/CmdStart.cs

```diff
@@ -54,6 +54,17 @@ public override async Task ExecuteArmed(ServerBattle battle, Say e)
             }
 
         }
+        public override int GetPollWinMargin(ServerBattle battle, int numVoters)
+        {
+            // Require one more vote to start a game with uneven teams so at least one player in the smaller
+            // team needs to agree to start the game. This is particularly relevant for the 2v1 case.
+            if (battle.Mode == PlasmaShared.AutohostMode.Teams) {
+                return 1 + numVoters % 2;
```

Comment:
The 1 is probably meant to be "whatever the base margin is"?
```suggestion
                return base.GetPollWinMargin(battle, numVoters) + numVoters % 2;
```

Reply:
Yup let's be consistent